aptos-move/e2e-move-tests/src/tests/existence_check.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{assert_success, tests::common, MoveHarness};
+use aptos_package_builder::PackageBuilder;
+use aptos_types::{
+    account_address::AccountAddress, move_utils::MemberId, on_chain_config::FeatureFlag,
+};
+use std::{path::Path, str::FromStr};
+
+fn run_transactions(features: Vec<FeatureFlag>, path: &Path, function_names: &[&str]) -> u64 {
+    let mut h = MoveHarness::new_with_features(features, vec![]);
```

Comment:
You should also explicitly disable your feature here, second vector, because if it is on by default the tests will not actually test 2 behaviours